fili-core/src/test/groovy/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProviderSpec.groovy

```diff
@@ -138,86 +132,31 @@ class LuceneSearchProviderSpec extends SearchProviderSpec<LuceneSearchProvider>
         1 * keyValueStore.put('cardinality_key', '14')
     }
 
-    def "moveDirEntries moves all entries of a directory to a new directory, while keeping all old empty dirs"() {
-        expect:
-        Files.exists(sourcePath)
-        Files.exists(file1)
-        Files.exists(file2)
-        Files.exists(file3)
-        Files.exists(subDir)
-        Files.exists(file4)
-
-        !Files.exists(destinationPath.resolve("segments_1"))
-        !Files.exists(destinationPath.resolve("_1.cfs"))
-        !Files.exists(destinationPath.resolve("_1.si"))
-        !Files.exists(destinationPath.resolve("subDir"))
-        !Files.exists(destinationPath.resolve("subDir").resolve("subDirFile"))
-
-        when:
-        searchProvider.moveDirEntries(sourceDir, destinationDir)
-
-        then:
-        Files.exists(sourcePath)
-        !Files.exists(file1)
-        !Files.exists(file2)
-        !Files.exists(file3)
-        Files.exists(subDir)
-        !Files.exists(file4)
-
-        and:
-        Files.exists(destinationPath.resolve("segments_1"))
-        Files.exists(destinationPath.resolve("_1.cfs"))
-        Files.exists(destinationPath.resolve("_1.si"))
-        Files.exists(destinationPath.resolve("subDir"))
-        Files.exists(destinationPath.resolve("subDir").resolve("subDirFile"))
-    }
-
-    def "deleteDir deletes all entries under a specified directory including the directory itself"() {
```

Comment:
Moved to `UtilsSpec.groovy` (`Utils.deleteFiles()` itself has not been tested so far)